src/components/diagrams/singleLineDiagram/single-line-diagram-content.tsx

```diff
@@ -471,10 +525,10 @@ function SingleLineDiagramContent(props) {
 
             // Rotate clicked switch while waiting for updated sld data
             if (locallySwitchedBreaker?.id) {
-                const breakerToSwitchDom = document.getElementById(locallySwitchedBreaker.id);
-                if (breakerToSwitchDom.classList.value.includes('sld-closed')) {
+                const breakerToSwitchDom: HTMLElement | null = document.getElementById(locallySwitchedBreaker.id);
+                if (breakerToSwitchDom && breakerToSwitchDom.classList.value.includes('sld-closed')) {
```

Comment:
Like this it's more readable
```suggestion
                if (breakerToSwitchDom?.classList.value.includes('sld-closed')) {
```

Reply:
done